70_Pushover.pm

```diff
@@ -235,80 +245,100 @@ sub Pushover_CGI() {
         # search for existing receipt
         keys %{ $hash->{READINGS} };
         while ( my ( $key, $value ) = each %{ $hash->{READINGS} } ) {
-            $revReadings{ $value->{VAL} } = $1
-              if ( defined( $value->{VAL} ) && $key =~ /^cb_(\d+)$/ );
+            if ( defined( $value->{VAL} ) && $key =~ /^cb_(\d+)$/ ) {
+                $revReadings{ $value->{VAL} } = $1;
+                $revReadings{ $1 } = $1;
+            }
         }
 
         if ( defined( $revReadings{$receipt} ) ) {
             my $rAct      = "cbAct_" . $revReadings{$receipt};
             my $rAck      = "cbAck_" . $revReadings{$receipt};
             my $rAckAt    = "cbAckAt_" . $revReadings{$receipt};
             my $rAckBy    = "cbAckBy_" . $revReadings{$receipt};
+            my $rAckByDev = "cbAckByDev_" . $revReadings{$receipt};
```

Comment:
3) The actual device that acknowledge the message is being recorded now - this is especially useful when sending to groups